src/component/page/bruker-detaljer/tildel-veileder-modal/TildelVeilederModal.tsx

```diff
@@ -118,71 +122,66 @@ export const TildelVeilederModal = (props: Props): React.ReactElement => {
 
   return (
     <>
-      <BaseModal
-        tittel={'Endre veiledere'}
+      <Modal
+        header={{ heading: 'Endre veiledere' }}
         open={props.open}
         onClose={handleClose}
         className={styles.modal}
       >
-        <Detail className={styles.detail}>
-          Tildel til{' '}
-          {lagBrukerNavn(
-            props.deltaker.fornavn,
-            props.deltaker.mellomnavn,
-            props.deltaker.etternavn
+        <Modal.Body className={styles.modalBody}>
+          <Detail className={styles.detail}>
+            Tildel til{' '}
+            {lagBrukerNavn(
+              props.deltaker.fornavn,
+              props.deltaker.mellomnavn,
+              props.deltaker.etternavn
+            )}
+          </Detail>
+          <UNSAFE_Combobox
```

Comment:
Er det mulig å fjerne en valgt veileder her? Jeg fikk ikke til det selv.

Reply:
Hvordan da tenker du? at de har en slags claer knapp? jeg kan bruke tastaturet til å fjerne, men kanskje flere måter vi hadde før?

Reply:
De hadde en clearButton, men den er deprecated så vet ikke om de skal få det inn eller ikke

Reply:
På forrige komponent var det en `X` knapp for å fjerne valgt veileder. Det funka med backspace ja, bare jeg som ble forvirra.

Reply:
Jeg ser at vanlig oppførsel, så fjernes ikke valget i lista og hvis man trykker på samme felt en gang til så fjernes den. Men vi filtrer lista så vi kan ikke velge en gang til, kanskje vi skal ha det sånn som er nå? eller må jeg fikse mye med hva som er lovlig valg